src/main/java/eiteam/esteemedinnovation/transport/fluid/screw/TileEntityPump.java

```diff
@@ -127,25 +127,27 @@ public void safeUpdate() {
             offsetPos = getOffsetPos(outputDir);
             if (myTank.getFluidAmount() > 0 && progress == 100) {
                 TileEntity tile = world.getTileEntity(offsetPos);
```

Comment:
These changes don't appear to be relevant to the Boiler?

Reply:
These changes were to make the screw functional enough to test the boiler. I had to test if the boiler still properly accepted fluid input, but didn't have anything that worked. I chose to fix this instead of adding another mod into my dev environment.

Reply:
Ah, that makes sense.